array-reduce/main.js

```diff
@@ -0,0 +1,41 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const account = [
+  { type: 'deposit', amount: 150 },
+  { type: 'deposit', amount: 20 },
+  { type: 'withdrawal', amount: 5 },
+  { type: 'deposit', amount: 100 },
+  { type: 'withdrawal', amount: 25 },
+  { type: 'withdrawal', amount: 60 }
+];
+
+const traits = [
+  { color: 'yellow' },
+  { type: 'electric' },
+  { name: 'pikachu' },
+  { level: 15 },
+  { trainer: 'ash' }
+];
+
+const sum = (previousValue, currentValue) => previousValue + currentValue;
+console.log(numbers.reduce(sum));
+
+const product = (previousValue, currentValue) => previousValue * currentValue;
+console.log(numbers.reduce(product));
```

Comment:
`sum` and `product` should be the final sum and product (respectively), not the callback functions.

---

![image](https://user-images.githubusercontent.com/16143771/137548554-2f06cf1d-b630-48d5-b2df-6fd8841be767.png)